adserver/tasks.py

```diff
@@ -588,3 +590,38 @@ def notify_of_publisher_changes(difference_threshold=0.25, min_views=10_000):
                             ),
                         },
                     )
+
+
+@app.task(time_limit=60 * 60 * 4)
+def archive_offers(day=None, delete=False):
+    """
+    Archive offers from a single day.
+
+    :arg day: An optional datetime object representing a day. If not specified, archive offers 91 days ago.
+    :arg delete: Delete archived offers IF they were successfully archived to settings.BACKUPS_STORAGE
+    """
+    if not day:
+        day = get_ad_day() - datetime.timedelta(days=91)
+
+    start_date, _ = _get_day(day)
+
+    args = [
+        "-s",
+        start_date.strftime("%Y-%m-%d"),
+        "-e",
+        start_date.strftime("%Y-%m-%d"),
+    ]
+    if delete:
+        args += ["-d"]
```

Comment:
I like using long-form args in scripts, so we can easily tell what it's doing:

```suggestion
        args += ["--delete-offers"]
```